plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -804,5 +804,29 @@ SecRule REQUEST_FILENAME "@contains /config/apps/ransomware_protection/extension
     ctl:ruleRemoveTargetById=932130;ARGS:value,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0'"
 
+#
+# [ Nextcloud Notes ]
+#
+
+# Note name and content can be anything
+SecRule REQUEST_URI "@rx ^\/index\.php\/apps\/notes\/api\/v[0-9]\/notes" \
+    "id:9508910,\
+    phase:1,\
+    pass,\
+    nolog,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.content,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.title"
+
+# Note name and content can be anything
+# PUT - when making changes/creating notes
+# DELETE - when notes are deleted
+SecRule REQUEST_URI "@beginsWith /apps/notes/notes/" \
+    "id:9508920,\
+    phase:1,\
+    pass,\
+    nolog,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.content,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.title,\
+    setvar:'tx.allowed_methods=GET HEAD POST OPTIONS PUT DELETE'"
```

Comment:
Rewrite this line like this:
`setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE"`